src.vba/SendInternalTransferForm.frm

```diff
@@ -21,7 +21,7 @@ End Sub
 Private Sub ConfirmButton_Click()
     'On Error Resume Next
     Dim myFile As String: myFile = PathBox.Value
-    Dim amount As Long: amount = getAmountLong(Utils.clearNonNumeric(AmountBox.Value))
+    Dim amount As Long: amount = getAmountLong(Utils.clearNonNumeric(AmountBox.Value) / 100)
```

Comment:
Cara, eu mudaria aqui. Não acho que tenhamos que usar a função getAmountLong. Acho que essa função tem que ser usada só qdo o impute for direto na planilha, e não no form. Eu colocaria assim:

`Dim amount As Long: amount = CLng(Utils.clearNonNumeric(AmountBox.Value))`